src/studio/src/designer/frontend/packages/schema-editor/src/components/Editor.test.tsx

```diff
@@ -272,14 +273,14 @@ test('should show context menu and trigger correct dispatch when adding a combin
 });
 
 test('should only be possible to add a reference to a combination type', async () => {
-  const [store] = await renderEditor({
+  const { store, user } = renderEditor({
```

Comment:
`store` is unused and can be removed
```suggestion
  const { user } = renderEditor({
```